solana.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 //go:generate go run genchains_solana.go
+//go:generate go run generate_all_selectors.go
```

Comment:
wouldn't this run the same logic seven times? because each chain file is doing go:generate..... ?

Reply:
Yeah I guess it's lesser evil of the two option. lgtm